src/test/kotlin/no/nav/syfo/behandler/api/BehandlerApiSpek.kt

```diff
@@ -97,6 +97,31 @@ class BehandlerApiSpek : Spek({
                             behandlerList.size shouldBeEqualTo 1
                         }
                     }
+                    it("search should exclude invalidated Behandler") {
+                        generateFastlegeResponse()
+                        val behandlerRef = with(
+                            handleRequest(HttpMethod.Get, url) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(validToken))
+                                addHeader(NAV_PERSONIDENT_HEADER, UserConstants.ARBEIDSTAKER_FNR.value)
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                            val behandlerList = objectMapper.readValue<List<BehandlerDTO>>(response.content!!)
+                            behandlerList.size shouldBeEqualTo 1
+                            UUID.fromString(behandlerList[0].behandlerRef)
```

Comment:
Denne brukes ikke til noe? 🤔 

Det er vel andre tester som sjekker at `get` funker, hadde det da holdt å bare gjøre ett kall i denne testen, såp man ser at invaliderte behandlere ikke blir med?

Reply:
Jo, det er denne som lagrer fastlegen som en instans av behandler (det er en sideeffekt av get-kallet), og så tilordnes behandlerRef til den opprettede behandleren.

Reply:
1. Men `UUID.fromString(behandlerList[0].behandlerRef)` gjør ingenting?
2. Man kunne lagret behandleren i databasen selv da 🤔

Reply:
Jo, UUID.fromString(behandlerList[0].behandlerRef) blir tilordnet til 
val behandlerRef.

Ja, enig i at det er andre måter å få lagret behandleren på, men dette er jo faktisk det som skjer i app'en (dvs det er slik fastlegene blir lagret som behandlere for en arbeidstaker).